### PR TITLE
v6.9.1

### PR DESCRIPTION
**Pull request by m4theushw**
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: packages/x-license-pro/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.9.0",
+  "version": "6.9.1",
```
Comment (LukasTy) [inline packages/x-license-pro/package.json:3]: This package only had a babel minor and `@mui/utils` patch version bumps, and they are not locked to specific version.
Do you think that it warrants a release? 🤔

Comment (LukasTy) [reply packages/x-license-pro/package.json]: Gotcha. 👌

**Review by richbustos — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,68 @@
...
+- 🌍 Add Chinese (Hong Kong) (zh-HK) locale on the pickers
+- 🌍 Improve Romanian (ro-RO) and Hungarian (hu-HU) translations on the pickers and the data grid
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### `@mui/x-data-grid@v6.9.1` / `@mui/x-data-grid-pro@v6.9.1` / `@mui/x-data-grid-premium@v6.9.1`
+
+#### Changes
+
+- [DataGrid] Add Joy UI tooltip and `loadingOverlay` slots (#9028) @cherniavskii
```
Comment (cherniavskii) [inline CHANGELOG.md:22]:
```suggestion
- [DataGrid] Add Joy UI `tooltip` and `loadingOverlay` slots (#9028) @cherniavskii
```

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,68 @@
...
+- [docs] Update charts overview.md (#9429) @brentertz
+
+### Core
+
+- [core] Add `edit-mode` to priority support action (#9483) @DanailH
+- [core] Fix priority support prompt action (#9472) @DanailH
+- [core] Update `uses` for priority support action (#9480) @DanailH
+- [core] Bumb update monorepo (#9476) @alexfauquette
+- [CHANGELOG] Revert "[CHANGELOG] Be clearer about each plan" @oliviertassinari
+- [CHANGELOG] Be clearer about each plan @oliviertassinari
```
Comment (oliviertassinari) [inline CHANGELOG.md:63]: These two cancels themselves, we could remove it.

```suggestion
```

**Review by romgrk — APPROVED** [review]

**Comment by flaviendelangle** [comment]
Can we please add #9526 and #9356 ?

**Comment by cherniavskii** [comment]
Please also include https://github.com/mui/mui-x/pull/9499 and https://github.com/mui/mui-x/pull/9516 which were merged today :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,71 @@
...
+- [l10n] Add Chinese (Hong Kong) (zh-HK) locale (#9468) @samchiu90
+- [l10n] Improve Romanian (ro-RO) translations (#9436) @noraleonte
+
+### `@mui/x-charts@6.0.0-alpha.1`
+
+#### Changes
+
+- [charts] Take responsive container from data grid (#9497) @alexfauquette
+- [charts] Update README.md (#9426) @alexfauquette
+- [charts] Fix typo and small refacto (#9526) @flaviendelangle
```
Comment (LukasTy) [inline CHANGELOG.md:48]:
```suggestion
- [charts] Fix typo and small refactor (#9526) @flaviendelangle
```

**Review by LukasTy — APPROVED** [review]

**Review by flaviendelangle — APPROVED** [review]

**Pull request merged**